banzai/qc/saturation.py

```diff
@@ -35,16 +35,18 @@ def do_stage(self, images):
             logs.add_tag(logging_tags, 'SATFRAC', saturation_fraction)
             logs.add_tag(logging_tags, 'threshold', self.SATURATION_THRESHOLD)
             self.logger.info('Measured saturation fraction.', extra=logging_tags)
-            saturated = saturation_fraction >= self.SATURATION_THRESHOLD
-            if saturated:
+            is_saturated = saturation_fraction >= self.SATURATION_THRESHOLD
+            qc_results = {'saturated.failed': is_saturated,
```

Comment:
As a general question, do we want to have a failed=false for every check or just only have a failed tag if it actually failed?

Reply:
The former (failed=false for every check). I would argue to keep it this way, since it indicates whether or not a check was performed at all.

Reply:
Ok.